input/pagecontent/terminology.md

```diff
@@ -8,72 +8,68 @@ This implementation guide does not define any unique value sets. The value sets
-- [Procedure](https://tx.ontoserver.csiro.au/fhir/ValueSet/procedure-1)
-- [RCPA SPIA Pathology Reporting](https://healthterminologies.gov.au/fhir/ValueSet/spia-pathology-reporting-1)
-- [Reason for Request](https://healthterminologies.gov.au/fhir/ValueSet/reason-for-request-1)
-- [Service Type](https://healthterminologies.gov.au/fhir/ValueSet/service-type-1)
-- [Smoking Status](https://healthterminologies.gov.au/fhir/ValueSet/smoking-status-1)
-
-**Value sets published in FHIR**
-- [AdministrativeGender](https://hl7.org/fhir/R4/valueset-administrative-gender.html)
-- [AllergyIntolerance Clinical Status Codes](https://hl7.org/fhir/R4/valueset-allergyintolerance-clinical.html)
-- [AllergyIntolerance Verification Status Codes](http://hl7.org/fhir/R4/valueset-allergyintolerance-verification.html)
-- [AllergyIntoleranceSeverity](http://hl7.org/fhir/R4/valueset-reaction-event-severity.html)
-- [Australian Indigenous Status](https://healthterminologies.gov.au/fhir/ValueSet/australian-indigenous-status-1)
-- [Condition/Diagnosis Severity](https://hl7.org/fhir/R4/valueset-condition-severity.html)
-- [ConditionCategoryCodes](https://hl7.org/fhir/R4/valueset-condition-category.html)
-- [ConditionClinicalStatusCodes](https://hl7.org/fhir/R4/valueset-condition-clinical.html)
-- [ConditionVerificationStatus](https://hl7.org/fhir/R4/valueset-condition-ver-status.html)
-- [DataAbsentReason](https://hl7.org/fhir/R4/valueset-data-absent-reason.html)
-- [Encounter Reason Codes](https://hl7.org/fhir/R4/valueset-encounter-reason.html)
-- [EncounterStatus](https://hl7.org/fhir/R4/valueset-encounter-status.html)
-- [EventStatus](https://hl7.org/fhir/R4/valueset-event-status.html)
-- [Immunization Status Codes](https://hl7.org/fhir/R4/valueset-immunization-status.html)
-- [LocationMode](https://hl7.org/fhir/R4/valueset-location-mode.html)
-- [LocationStatus](https://hl7.org/fhir/R4/valueset-location-status.html)
-- [Medication request intent](https://hl7.org/fhir/R4/valueset-medicationrequest-intent.html)
-- [Medicationrequest status](https://hl7.org/fhir/R4/valueset-medicationrequest-status.html)
-- [NameUse](https://hl7.org/fhir/R4/valueset-name-use.html)
-- [Observation Category Codes](https://hl7.org/fhir/R4/valueset-observation-category.html)
-- [Observation Interpretation Codes](https://hl7.org/fhir/R4/valueset-observation-interpretation.html)
-- [Observation Reference Range Meaning Codes](https://hl7.org/fhir/R4/valueset-referencerange-meaning.html)
-- [ObservationStatus](http://hl7.org/fhir/R4/valueset-observation-status.html)
-- [Participant type](https://hl7.org/fhir/R4/valueset-encounter-participant-type.html)
-- [Provenance activity type](https://hl7.org/fhir/R4/valueset-provenance-activity-type.html)
-- [Vital Signs Units](https://hl7.org/fhir/R4/valueset-ucum-vitals-common.html)
-- [VitalSigns](https://hl7.org/fhir/R4/valueset-observation-vitalsignresult.html)
+|ValueSet|Profile where used in|Published|
```

Comment:
For 'Profile where used in', suggest 'Profile where ValueSet used'.

Reply:
Changed to "Profile where used" to be consistent with wording choices in CodeSystem table in AU Core.